dockhand-lru-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "dockhand-lru-registry.name" . }}
+  labels:
+    {{- include "dockhand-lru-registry.labels" . | nindent 4 }}
+spec:
+  serviceName: {{ include "dockhand-lru-registry.name" . }}
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "dockhand-lru-registry.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "dockhand-lru-registry.selectorLabels" . | nindent 8 }}
+      annotations:
+        checksum/config-maps: {{ (.Files.Glob "files/**").AsConfig | sha256sum }}
+    spec:
+      initContainers:
+        - name: init-registry-proxy
+          image: "{{ .Values.registry.image.repository }}:{{ .Values.registry.image.tag }}"
+          imagePullPolicy: {{ .Values.registry.image.pullPolicy }}
+          command:
+            - cp
+            - /bin/registry
+            - /registry/bin/registry
+          args:
+          volumeMounts:
+            - name: registry-bin
+              mountPath: /registry/bin
+      containers:
+        - name: cache-lru-proxy
+          image: "{{ .Values.proxy.image.repository }}:{{ .Values.proxy.image.tag }}"
+          imagePullPolicy: {{ .Values.proxy.image.pullPolicy }}
+          args:
+            - start
+            - --use-forwarded-headers
+            - --target-percentage
+            - {{ .Values.proxy.cleanSettings.targetUsagePercentage | quote}}
+            - --clean-tags-percentage
+            - {{ .Values.proxy.cleanSettings.cleanTagPercentage | quote}}
+            - --registry-dir
+            - /var/lib/registry
+            - --timezone
+            - {{ .Values.proxy.cleanSettings.timezone | quote }}
+            - --cleanup-cron
+            - {{ .Values.proxy.cleanSettings.cron | quote }}
+            - --registry-host
+            - 127.0.0.1:5000
+            - --port
+            - "80"
+            {{- if .Values.proxy.debug }}
+            - --debug
+            {{- end }}
+          ports:
+            - name: cache
+              containerPort: 80
```

Comment:
would it be better to use a high port to avoid confusion?  so a TLS server doesn't wind up on port 80?  also would allow for a non-root user.